spec.html

```diff
@@ -24176,16 +24314,265 @@ <h1>Runtime Semantics: Evaluation</h1>
     </emu-clause>
   </emu-clause>
 
+  <emu-clause id="sec-decorators">
+    <h1>Decorators</h1>
+    <h2>Syntax</h2>
+    <emu-grammar type="definition">
+      DecoratorList[Yield, Await] :
+        DecoratorList[?Yield, ?Await]? Decorator[?Yield, ?Await]
+
+      Decorator[Yield, Await] :
+        `@` DecoratorMemberExpression[?Yield, ?Await]
+        `@` DecoratorParenthesizedExpression[?Yield, ?Await]
+        `@` DecoratorCallExpression[?Yield, ?Await]
+
+      DecoratorMemberExpression[Yield, Await] :
+        IdentifierReference[?Yield, ?Await]
+        PrivateIdentifier
+        DecoratorMemberExpression[?Yield, ?Await] `.` IdentifierName
```

Comment:
```suggestion
        DecoratorMemberExpression[?Yield, ?Await] `.` IdentifierName
        DecoratorMemberExpression[?Yield, ?Await] `.` PrivateIdentifier
```

Otherwise we would allow invalid `@#dec` and disallow valid `@C.#dec`.

Reply:
Yes that's correct I think, and given `@#dec` does not have any logical meaning I don't think this would be a normative change, so going to update the spec with the fix.